tests/cypress/support/commands.js

```diff
@@ -217,10 +217,17 @@ Cypress.Commands.add('closeSettings', () => {
         });
 });
 
-Cypress.Commands.add('changeAnnotationMode', (mode) => {
+Cypress.Commands.add('changeAnnotationMode', (mode, labelName) => {
```

Comment:
I would suggest renaming to ``changeWorkspace``.

Reply:
Renamed.